gateway/src/main/scala/filodb/gateway/GatewayServer.scala

```diff
@@ -58,6 +59,9 @@ import filodb.timeseries.TestTimeseriesProducer
 object GatewayServer extends StrictLogging {
   Kamon.init
 
+  val MAX_RECORD_TAG_VALUE_SIZE_BYTES = 16384  // 2^14; 2^15 is lucene limit
+  val RECORD_TAG_VALUE_BYTES_PER_CHAR = 2;
```

Comment:
Any better way to do this? Doesn't look like a `Charset.bytesPer` exists.
Also: this isn't necessarily "2"-- I still need to look through the agent code to find the encoding.